include/exadg/incompressible_navier_stokes/time_integration/time_int_bdf.h

```diff
@@ -132,6 +140,39 @@ class TimeIntBDF : public TimeIntBDFBase<Number>
   std::shared_ptr<HelpersALE<Number> const> helpers_ale;
 
 private:
+  template<typename Lambda>
```

Comment:
I think one should generally introduce a template parameter only if the type is not known (in the sense of generic programming). I think this is not given here, so that the template parameter should be avoided here.

Reply:
The only way to avoid this is by using a `std::function` as I did in [1f39cef](https://github.com/exadg/exadg/pull/458/commits/1f39cef8d688232b73b239620a45f409a169e862). The disadvantage is performance as far as I remember. Anyway, in this part of the code it does not matter.